src/main/java/dk/bec/bookanything/model/FacilityTypeEntity.java

```diff
@@ -1,25 +1,30 @@
 package dk.bec.bookanything.model;
 
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
-import java.util.List;
-import java.util.UUID;
 
 @Data
 @Table
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
 @Entity(name = "facility_type")
 public class FacilityTypeEntity {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private Long id;
 
-    @Column
-    @GeneratedValue(generator = "uuid2")
-    private UUID uuid;
-
     @Column(length = 200)
     private String name;
+
+
+    public FacilityTypeEntity(String name) {
```

Comment:
How this object is allowed to exists without `id` set?
All in all, the fact that you had to add 3 constructors and 1 builder shows that there is something wrong with design of this entity or perhaps it is used as DTO, Entity and Api representation model at the same time?